src/classes/Ingredient.js

```diff
@@ -1,12 +1,15 @@
+import ingredientsData from '../data/ingredients-test-data.js';
+
 class Ingredient {
-  constructor(id, name, cost) {
+  constructor(id, quantity) {
     this.id = id;
-    this.name = name;
-    this.estimatedCost = cost;
+    this.name = ingredientsData.find(ingredient => this.id === ingredient.id).name;
+    this.estimatedCostInCents = ingredientsData.find(ingredient => this.id === ingredient.id).estimatedCostInCents;
```

Comment:
Linking the data from the ingredients data to the ingredient class and finding by id (when instantiated from the recipe class)